src/test/javascript/spec/service/orion/orion-assessment.service.spec.ts

```diff
@@ -111,48 +107,53 @@ describe('OrionAssessmentService', () => {
 
         testConversion(mockReader as any);
 
-        expect(alertErrorStub).to.have.been.calledOnceWithExactly('artemisApp.assessmentDashboard.orion.downloadFailed');
+        expect(alertErrorStub).toHaveBeenCalledTimes(1);
+        expect(alertErrorStub).toHaveBeenCalledWith('artemisApp.assessmentDashboard.orion.downloadFailed');
     });
 
     /**
      * Helper to test the conversion with the fileReader
      * @param mockReader mock reader to test
      */
     function testConversion(mockReader: FileReader) {
-        const isCloningSpy = spy(orionConnectorService, 'isCloning');
-        const exportSubmissionStub = stub(programmingAssessmentExportService, 'exportReposByParticipations');
-        const lockAndGetStub = stub(programmingSubmissionService, 'lockAndGetProgrammingSubmissionParticipation');
-        const readerStub = stub(window, 'FileReader');
+        const isCloningSpy = jest.spyOn(orionConnectorService, 'isCloning');
+        const exportSubmissionStub = jest.spyOn(programmingAssessmentExportService, 'exportReposByParticipations');
+        const lockAndGetStub = jest.spyOn(programmingSubmissionService, 'lockAndGetProgrammingSubmissionParticipation');
+        const readerStub = jest.spyOn(window, 'FileReader');
```

Comment:
I'd suggest setting the stubs here directly => inline

Reply:
In this case I disagree, the names are long enough so it will end up in two lines anyways, and then it might as well be sorted nicely with the definitions separeted from the mocked values

Reply:
I see your point. I'd suggest inlining the readerStub and moving it up to the spy so the other stubs can be next to their stubbing statements.

In this case consistency doesn't make it easier to read since it's just reading through more LoC